reporter-web-app/src/components/RuleViolationsTable.js

```diff
@@ -156,6 +146,16 @@ const RuleViolationsTable = (props) => {
                         </span>
                     )
             ),
+            sorter: (a, b) => {
+                if (a.severityIndex < b.severityIndex) {
```

Comment:
Could we eliminate the code branches by just doing `return a.severityIndex - b.severityIndex` ?

Reply:
Added a separate commit to make this a one liner.